.github/workflows/ci_tests.yaml

```diff
@@ -40,14 +40,14 @@ jobs:
             isDraft: true
           - os: windows-latest
             isDraft: true
-        # Pair Python 3.8 with NumPy 1.20 and Python 3.10 with NumPy 1.22
-        # Only install optional packages on Python 3.10/NumPy 1.22
+        # Pair Python 3.8 with NumPy 1.20 and Python 3.10 with NumPy 1.23
+        # Only install optional packages on Python 3.10/NumPy 1.23
         include:
           - python-version: '3.8'
             numpy-version: '1.20'
```

Comment:
Question. Do we want to just set our minimum tested NumPy version to 1.21 in the CI since `xarray` doesn't work with Numpy 1.20 and they won't be releasing a patch as per https://github.com/pydata/xarray/issues/6818#issuecomment-1215278493? Due date to drop Numpy 1.20 is Jan 31, 2023 (5 months) per https://numpy.org/neps/nep-0029-deprecation_policy.html#drop-schedule.

Reply:
I think setting numpy to v1.21 makes sense. I'm not sure what the plan is for the pygmt v0.8 release, but I don't think we'll have 2 releases before January 2023, so our next release should be up to date on minimum versions.

Reply:
Cool. Just to be clear, the suggesting is to just set NumPy 1.21 for the **Continuous Integration (CI)** so that the unit tests pass. We shouldn't change the minimum supported NumPy mentioned in https://github.com/GenericMappingTools/pygmt/tree/v0.7.0#compatibility-with-gmtpythonnumpy-versions until after Jan 2023 (though users using NumPy 1.20 will need to figure out the `xarray` dependency themselves).